frontend/src/views/pages/DashboardPage.tsx

```diff
@@ -1,27 +1,36 @@
 import { Typography, Stack, Box } from "@mui/material";
 import { useLocation } from "react-router-dom";
-import Overview from "../components/Dashboard/Overview/Overview";
 import GraphGrid from "../components/Dashboard/GraphGrid/GraphGrid";
-import BiasScore from "../components/Dashboard/BiasScore/BiasScore";
+import Overview from "../components/Dashboard/Overview/Overview";
+import LetterGrade from "../components/Dashboard/LetterGrade/LetterGrade";
+import Footer from "../components/Footer/Footer";
 
 function DashboardPage() {
   const location = useLocation();
   const { dashboardData } = location.state;
 
   return (
+    <>
     <Stack>
-      <Typography variant="body2" pb={3}>
-        {dashboardData.filePath}
+      <Typography tabIndex={0} aria-label={`File path displayed as ${dashboardData.filePath.split('/').pop()}`} p={14} sx={{textAlign:'left', fontFamily: 'Montserrat', color: "#9921D2", fontWeight: 500, fontSize: '15pt'}} pb={3}>
```

Comment:
I'm not too sure if we should keep this! I know it is in the Figma design but I recall in tutorial the look of this was sort of questioned, since I think the user can only upload 1 file anyways it might be unnecessary.

But I think it is fine to keep as well!

Reply:
Modified this to be under a title with text 'Dashboard' so it looks more cohesive!